pyccel/codegen/printing/cwrappercode.py

```diff
@@ -449,6 +556,44 @@ def _body_management(self, used_names, variable, collect_var, default_value = No
 
         return body, tmp_variable
 
+    def untranslatable_function(self, wrapper_name, wrapper_args, wrapper_results, error_msg):
+        """
+        Certain functions are not handled in the wrapper (e.g. private),
+        This creates a wrapper function which raises NotImplementedError
+        exception and returns NULL
+
+        Parameters
+        ----------
+        wrapper_name    : string
+            The name of the C wrapper function
+
+        wrapper_args    : list of Variables
+            List of python object variables
+
+        wrapper_results : Variable
+            python object variable
```

Comment:
Shouldn't we say that `wrapper_args` contains the function arguments and `wrapper_results` the function results?

By the way, given line 591 below I think that `wrapper_results` is a list.

Reply:
Fixed : c21c497